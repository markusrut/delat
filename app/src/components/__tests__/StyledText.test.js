import renderer from "react-test-renderer";
import { MonoText } from "../StyledText";

describe("StyledText", () => {
  const monoComponent = <MonoText>testing mono text!</MonoText>;
  it("renders correctly", () => {
    const tree = renderer.create(monoComponent).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
