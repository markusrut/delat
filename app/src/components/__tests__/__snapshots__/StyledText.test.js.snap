// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledText renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "fontFamily": "space-mono",
        },
      ],
    ]
  }
>
  testing mono text!
</Text>
`;
